wagtail_wordpress_import/constants.py

```diff
@@ -72,176 +75,41 @@
 FILTER_MAPPING = {
     "bold": [
         # output content in <b></b>
-        "color: rgb(0, 0, 153); font-weight: bold;",
-        "color: rgb(0, 102, 0); font-weight: bold;",
-        "color: rgb(204, 0, 0); font-weight: bold;",
-        "font-style: italic; font-weight: bold;",
-        "FONT-WEIGHT: bold",
-        "font-weight: bold;",
-        "font-weight: bold; color: #006600;",
-        "font-weight: bold; color: #990000;",
-        "font-weight: bold; color: #cc0000;",
-        "font-weight: bold; color: #cc0000; font-style: italic;",
-        "font-weight: bold; color: #ff0000;",
-        "font-weight: bold; color: rgb(0, 102, 0);",
-        "font-weight: bold; color: rgb(153, 0, 0);",
-        "font-weight: bold; color: rgb(153, 0, 0); font-style: italic;",
-        "font-weight: bold; color: rgb(204, 0, 0);",
-        "font-weight: bold; color: rgb(204, 0, 0); font-style: italic;",
-        "font-weight: bold; font-size: 100%;",
-        "font-weight: bold; font-size: 130%;",
-        "font-weight: bold; font-size: 180%;",
-        "font-weight: bold; font-style: italic;",
-        "font-weight: bold; font-style: italic; color: #990000;",
-        "font-weight: bold; font-style: italic; color: rgb(153, 0, 0);",
-        "font-weight: bold;font-size:100%;",
-        "font-weight: bold;font-size:130%;",
-        "FONT-WEIGHT: bold;font-size:130%;",
-        "FONT-WEIGHT: bold;font-size:180%;",
-        "font-weight: bold;font-size:180%;",
+        "font-weight:bold;",
     ],
     "italic": [
         # output content in <i> tag
-        "color: #990000; font-style: italic;",
-        "color: rgb(0, 0, 0); font-style: italic;",
-        "color: rgb(102, 0, 0); font-style: italic;",
-        "FONT-STYLE: italic",
-        "font-style: italic;",
-        "font-style: italic; color: #990000;",
-        "font-style: italic; color: #ffff00;",
-        "font-style: italic; color: rgb(153, 0, 0);",
-        "font-style: italic; color: rgb(204, 0, 0);",
-        "font-style: italic; color: rgb(255, 255, 0);",
-        "font-style: italic; font-size: 100%;",
-        "font-style: italic; font-size: 85%;",
-        "font-style: italic;font-size:100%;",
-        "font-style: italic;font-size:130%;",
-        "font-style: italic;font-size:85%;",
+        "font-style:italic;",
     ],
     "bold-italic": [
-        "font-style: italic; font-weight: bold;",
-        "font-weight: bold; font-style: italic;",
+        "font-style:italic; font-weight:bold;",
```

Comment:
What if there's another style inserted in between?

Reply:
#40